fundamentals-recap/problem-set-conditionals-recap.md

```diff
@@ -492,13 +500,15 @@ else:
 
 * 
 ``` Python
-if customer_age <= 10:
+if customer_age < 0:
```

Comment:
Whatever change is applied to the correct answer above should be mirrored here (either flipping => to >= or changing to an else).